src/plays/emojipedia/Emojipedia.js

```diff
@@ -32,8 +32,7 @@ function Emojipedia(props) {
   useEffect(() => {
     const delayFn = setTimeout(() => {
       if (query) {
-        setEmojisList((prevEmojiList) =>
-          prevEmojiList?.filter((emoji) =>
+        setEmojisList(data.filter((emoji) =>

```

Comment:
i see more than 2.5k emojis are coming from the api. please use `useTransition` hook here to make the filter.

Reply:
Okay working on that, I have not used useTransition before, but as I read its kind of loading, will make the necessary changes and upload

Reply:
@Angryman18 can you please review the code again I have made the changes suggested

Reply:
on deployed url i still see the page break and api call error. although if the api is not meant to give data on the deployed url then handle the error on api fail
![Screenshot from 2022-10-17 12-27-22](https://user-images.githubusercontent.com/63530626/196109474-7f35e632-4ef1-4af9-a3a8-f03d475da148.png)